### PR TITLE
Grid Filtering - Fix WC/Blazor grid snippets and sections. Add styling.

### PR DESCRIPTION
**Pull request by MayaKirova**

### CODE REVIEW

File: doc/en/components/grids/_shared/filtering.md [hunk 1]
```diff
@@ -202,11 +207,13 @@ gridFilteringExpressionsTree.filteringOperands.push(priceFilteringExpressionsTre
 this.@@igObjectRef.filteringExpressionsTree = gridFilteringExpressionsTree;
 ```
 
+<!-- end:Angular -->
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/_shared/filtering.md:210]: missing space after "end:"

File: doc/en/components/grids/_shared/filtering.md [hunk 2]
```diff
@@ -408,15 +426,16 @@ export class BooleanFilteringOperand extends IgxBooleanFilteringOperand {
 }
 ```
 
+<!-- end:Angular -->
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/_shared/filtering.md:429]: missing space after "end:"

File: doc/en/components/grids/_shared/filtering.md [hunk 3]
```diff
@@ -545,12 +566,16 @@ constructor() {
...
 
-<!-- Angular -->
+<!-- ComponentEnd: HierarchicalGrid -->
+
 
-<!-- NOTE this sample is differed -->
 
 `sample="/{ComponentSample}/custom-filtering", height="600", alt="{Platform} {ComponentTitle} Custom Filtering Example"`
 
+<!-- end:Angular, WebComponents -->
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/_shared/filtering.md:575]: missing space after "end:"

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Comment by mddragnev** [comment]
The first sample that is used in the docs is filtering-options and it shows excel style filtering instead of quick filtering. Probably not a big deal but I think in the filtering topic should be quick filtering

**Comment by MayaKirova** [comment]
@mddragnev Sample mode updated here: https://github.com/IgniteUI/igniteui-xplat-examples/pull/199
Also updated topic to match.

**Review by mddragnev — APPROVED** [review]

**Comment by dkamburov** [comment]
@HUSSAR-mtrela can you please do a review again

**Review by dkamburov — APPROVED** [review]

File: doc/en/components/grids/_shared/filtering.md [hunk 4]
```diff
@@ -145,6 +145,8 @@ The filtering feature is enabled for the `{ComponentName}` component by setting
 > [!Note]
 > If values of type *string* are used by a column of data type *date*, the `{ComponentName}` won't parse them to *date* objects and using filtering conditions won't be possible. If you want to use *string* objects, additional logic should be implemented on the application level, in order to parse the values to *date* objects.
 
+<!-- Angular, WebComponents -->
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/_shared/filtering.md:148]: there is no end tag for this build flag

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
see my comment

File: doc/en/components/grids/_shared/filtering.md (hunk 4 above)
Comment (MayaKirova) [reply doc/en/components/grids/_shared/filtering.md:148]: Thanks, second one has been removed.

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**